tests/containers/helm.pm

```diff
@@ -0,0 +1,73 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test deploy a helm chart
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use utils qw(zypper_call script_retry);
+use version_utils qw(is_sle);
+use registration qw(add_suseconnect_product get_addon_fullname);
+use mmapi 'get_current_job_id';
+
+sub run {
+    my ($self) = @_;
+
+    $self->select_serial_terminal;
+    $self->{deployment_name} = "apache-" . get_current_job_id();
+    my $chart = "bitnami/apache";
+
+    zypper_call("in -y jq aws-cli helm kubernetes-client");
+
+    # Check helm version
+    record_info('Helm version', script_output("helm version"));
+
+    # Connection to AWS
+    my $provider = $self->provider_factory(service => 'EKS');
+    $self->{provider} = $provider;
+
+    # Access to Cluster
+    record_info('Kubectl version', script_output("kubectl version"));
+    assert_script_run("cat \$HOME/.kube/config");
+    assert_script_run("kubectl get nodes");
+
+    # Ensure is deleted
+    script_run("helm delete $self->{deployment_name}");
+
+    # Add repo, search and show values
+    assert_script_run("helm repo add bitnami https://charts.bitnami.com/bitnami");
+    assert_script_run("helm search repo apache");
```

Comment:
Just out of curiosity, should not the test try `helm repo update` before search and install ?

Reply:
yes, we can add this. Thanks

Reply:
done and testing